### DISCUSSION TITLE
Set *geodetic* latitude position from python for initial condition (not via xml)

### DISCUSSION THREAD

**Discussion by clolsonus** [JSBSim-Team/jsbsim#1069] [General]
Hey everyone,

I have been messing around on the side a bit with jsbsim and the python wrappers.  I can start up anywhere I like using an IC.xml file, but I'd like to explore setting "ic/xyz" properties programmatically.  From what I can see, the "ic/xyz" properties expect geocentric coordinates, and FGInitialCondition or FGLocation is not exposed to the python api.  Is there a quick way to do this or have gone down a rabbit hole no one ever anticipated?

Thanks!

Curt. (of FlightGear infamy) :-)

**Comment by seanmcleod** [comment]
Looks to me like both geocentric and geodetic options are available?

https://github.com/JSBSim-Team/jsbsim/blob/b9bfc2350e0e00e53246b65979cc18df2922a88d/src/initialization/FGInitialCondition.cpp#L1488-L1490

https://github.com/JSBSim-Team/jsbsim/blob/b9bfc2350e0e00e53246b65979cc18df2922a88d/src/initialization/FGInitialCondition.cpp#L1584-L1586

**Reply by clolsonus** [reply]
Ok, maybe it was just missed in the docs then ... I will give it a try.  Thanks for the quick reply!

**Comment by seanmcleod** [comment]
The source code is the definitive document 😉
